.github/workflows/developer.yml

```diff
@@ -17,19 +17,18 @@ jobs:
   developer:
     runs-on: ubuntu-latest
     env:
-      FC: gfortran-9
-      CC: gcc-9
```

Comment:
Do we actually need these to be set? Seems like we're building everything with MPI wrappers anyway.

Reply:
Good question!